robot_tests/setup.py

```diff
@@ -0,0 +1,18 @@
+import setuptools
+
+setuptools.setup(
+    name="robottests",
```

Comment:
We should make the Python package name specific to our use-case; `robottests` could apply to anyone's robot framework test implementation.  How about `upgrade_testing_framework_robot`, or something?

Reply:
My thinking here is that I don't see the robot tests being directly tied to the UTF. No idea if we want to go this direction but I could potentially see something like an "assessment" tag being added to a set of tests or someone running certain tests on their own cluster. Will agree this package name is a bit vague though and maybe something like `robot_cluster_tests` or `robot_upgrade_tests` (even this is a bit leading) is better

Reply:
I'm fine w/ whatever name as long as it is more specific to what it's doing.  `robot_tests_cluster_upgrade`?